bahmni_api_feed/models/order_save_service.py

```diff
@@ -398,7 +395,13 @@ def _create_sale_order_line_function(self, sale_order, order):
                     uom = prod_obj.uom_id.id
                 )
                 price = self.env['account.tax']._fix_tax_included_price_company(sale_line._get_display_price(), prod_obj.taxes_id, sale_line.tax_id, sale_line.company_id)
-                
+                if prod_lot != None:
```

Comment:
This condition can be simplified and added as a single if block 
`if prod_lot != None and bool(self.env['ir.config_parameter'].sudo().get_param('bahmni_sale.sale_price_markup')) == True:`